src/fluree/db/dbfunctions/core.cljc

```diff
@@ -111,12 +117,46 @@
      :spec   spec
      :code   nil}))
 
+#?(:cljs
+   (defn- build-fn
+     [var fun]
+     (eval `(fn [~var]
+              ~fun))))
 
 (defn find-fn
   ([db fn-name]
    (find-fn db fn-name nil))
   ([db fn-name funType]
-   #?(:cljs (throw (ex-info "DB functions not yet supported in javascript!" {}))
+   #?(:cljs (cond
+              (identical? "nodejs" cljs.core/*target*)
+              (go-try
+                (let [forward-time-travel-db? (:tt-id db)]
+                  (or (if-not forward-time-travel-db? (get @db-fn-cache [(:network db) (:dbid db) fn-name]))
+                      (let [res (if-let [local-fn (get default-fn-map (symbol fn-name))]
+                                  (resolve-local-fn local-fn)
+                                  (let [query       {:selectOne ["_fn/params" "_fn/code" "_fn/spec"]
+                                                     :from      ["_fn/name" (name fn-name)]}
+                                        res*        (<? (dbproto/-query db query))
+                                        _           (if (empty? res*)
+                                                      (throw (ex-info (str "Unknown function: " (pr-str fn-name))
+                                                                      {:status 400
+                                                                       :error  :db/invalid-fn})))
+                                        params      (read-str (get res* "_fn/params"))
+                                        code        (<? (resolve-fn db (read-str (get res* "_fn/code")) funType params))
+                                        spec        (get res* "_fn/spec")
+                                        params'     (mapv (fn [x] (symbol x)) params)
```

Comment:
minor, but you can combine these to lines like this

```clojure
params' (->> params
             (mapv (fn [x] (symbol x)))
             (cons '?ctx)
             (into []))
```
again, this is minor and subject to tastes, so I'm only presenting it here as a suggestion. Take it or leave it.

Reply:
made the suggested change.  :) Ran tests to validate